exec/java-exec/src/main/java/org/apache/drill/exec/vector/complex/AbstractMapVector.java

```diff
@@ -173,7 +173,7 @@ public ValueVector getChildByOrdinal(int id) {
    *
    * Note that this method does not enforce any vector type check nor throws a schema change exception.
    */
-  protected void putChild(String name, ValueVector vector) {
+  public void putChild(String name, ValueVector vector) {
     putVector(name, vector);
```

Comment:
I'm not sure what it means for what is loaded to be right. But I went ahead and moved the UnionVector into the same package as AbstractMapVector, so I can use this method without making it public.